project-management/1_to_1_Sprint_User_Review_sessions_guidelines.md

```diff
@@ -0,0 +1,32 @@
+# 1 to 1 Sprint User Review 
+
+A 1 to 1 review of each Sprint’s activity will be, completed at the end of each Sprint.  
+With one member of the Fujitsu Team and one member of the Autistic community chosen by Autistica. 
+The review meetings can be undertaken via skype / or as a phone call with shared screens or in person dependant on the expressed preference of the member from the Autistic Community. 
+Autistica will source the member from the autistic community and ensure all consent required by Alan Turing is obtained. The community members will be selected from as broad a cross-section of the community as possible. As a result, the review process will be accommodating to their needs. 
+Autistica will give notice to Fujitsu in advance of who will be attending each 1 to 1 and how they would prefer for it to be, carried out to allow adequate time to plan. This will be in the form of a member booking form. (doc to be created) 
+
+## Prior to Session:
+
+The members of the autistic community should watch demonstration videos & look at the sprint review report from previous sprints to be up to date with project and what is to be asked of them.
+Autistica will ensure that all volunteers understand that this is a pre-requisite to involvement in the project, and will take responsibility to inform volunteers.
+
+## During the Session:
+
+The Fujitsu team member will record the session (if it is on skype) and make notes throughout. 
+The member of the autistic community will be walked through the sprint, starting with a description of the backlog and what has and hasn’t been achieved. 
+The member of the autistic community will be shown the platform so far and will be given space to comment, ask questions and give feedback on the platform. 
+All feedback will be recorded by the Fujitsu team member.
+
+## Post Session - Feedback / Findings :
+
+The Fujitsu team member will write up the notes from the meeting in the form of finding’s which will then be reviewed, prioritised and converted into user stories as decided by the project owner. 
+These user stories will be shared, with the autistic participant, to ensure the user stories are a true representation of the user’s point of view.   
+The member of the autistic community will be given a fortnight to agree the user stories are a true representation of the users point of view. 
+At this point the member of the autistic community can also provide feedback on the structure of sessions to iteratively improve the sessions. 
+
+## Approval:
+
+Once approval has been granted, from the Project Owner and Autistica/Turning and Fujitsu.
```

Comment:
```suggestion
Once approval has been granted from the Project Owner and Autistica/Turing and Fujitsu.
```

Who is the project owner?

Reply:
Product owner is Thomas, apologies, I'm still getting my head around the terminology